.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Probably not a great idea. We should not keep such files in the repo.

Reply:
Hmmm... I'm not a VSCode user. I can go download VSCode and try to setup our repo with it, but from googling, it seems like this file is needed for flow https://code.visualstudio.com/docs/languages/javascript#_can-i-use-other-javascript-tools-like-flow

Reply:
I had to configure this myself too when setting up the local env yesterday. Not sure if committing it here is the best approach (I've done so in my projects), but it's gonna be needed for every VSC user.